docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -18,13 +18,40 @@ These functions are described in more detail in the `PostgreSQL docs
 
 General-purpose aggregation functions
 =====================================
+``OrderableAgg``
+----------------
+
+.. class: OrderableAgg(expression, ordering=None, **extra)
```

Comment:
Is this class meant for users to extend or use directly? If not, it doesn't need to be documented.